src/main/java/io/seqera/migtool/MigTool.java

```diff
@@ -447,7 +462,12 @@ private int migrate(MigRecord entry) throws SQLException {
     }
 
     protected boolean checkMigrated(MigRecord entry) {
-        String sql = "select `id`, `checksum`, `script` from " + MIGTOOL_TABLE + " where `rank` = ? and `script` = ?";
+        String sql;
+        if(dialect.isPostgres() || dialect.isTestContainersPostgres()) {
+            sql = "select id, checksum, script from " + MIGTOOL_TABLE + " where rank = ? and script = ?";
+        } else {
+            sql = "select `id`, `checksum`, `script` from " + MIGTOOL_TABLE + " where `rank` = ? and `script` = ?";
+        }
```

Comment:
Why does Postgres require special handling?

Reply:
I added in the PR description.

> Backticks are a non-standard used by MySQL. Postgres does not support them by default, so the simple option was to add a conditional to check for the driver type and remove the backticks accordingly.